src/constants/Constants.js

```diff
@@ -197,3 +198,6 @@ export const DATA_TYPES = {
   incident: 'incident'
 };
 
+export const MARKDOWN_ARTIFACTS_DEFAULT_SERVER_ADDRESS = 'http://localhost:10888';
```

Comment:
What is this address?

Reply:
this is the default markdown artefact server that will be running in the same docker container.
you can see its implementation here: https://github.com/demisto/content/pull/14508

Reply:
It is too specific for the product, this is open source project, so better to provide it in the product itself.

Reply:
well... i do pass it as an argument from the script to reportService.
this is just a fallback default.

Reply:
i can remove the default.... and only rely on the argument received by the script

Reply:
Let's remove.